apps/site/src/components/layout/Footer.astro

```diff
@@ -0,0 +1,16 @@
+---
+const year = new Date().getFullYear();
+---
+
+<footer class="border-t border-t-gray-200 mt-auto">
+  <div class="max-w-7xl mx-auto p-2">
+    <p>
+      &copy; <span id="footer-year">{year}</span> - Code under MIT. Content (images
+      and articles) under CC BY-NC 4.0.
+    </p>
+  </div>
+</footer>
+
+<script is:inline>
+  document.getElementById("footer-year").innerText = new Date().getFullYear();
+</script>
```

Comment:
FYI: I'll have to validate this, but I think this is not 100% legal under German law :thinking:, let's keep it this way for now.